index.html

```diff
@@ -329,7 +329,14 @@
 It will continue to evolve and the task force will issue updates as often as needed. At the
 conclusion of the task force, the TAG intends to adopt this document as a Finding.
 
-<aside class="note">
+</section>
+
+# How This Document Fits In
```

Comment:
I like this. I looked it up, apparently I added support in ReSpec for "introductory" sections that serve as front matter. It's [just a matter of adding `class="introductory"`](https://github.com/w3c/respec/wiki/introductory). WDYT?

Reply:
I was looking for a way to avoid numbering this section, so SG. Oddly, I have to use `##` to keep this an `<h2>` if I put it into a `<section class="introductory">`.